test/e2e/kubectl.go

```diff
@@ -298,7 +298,11 @@ var _ = framework.KubeDescribe("Kubectl client", func() {
 
 			// Build the static kubectl
 			By("Finding a static kubectl for upload")
-			testStaticKubectlPath, err := findBinary("kubectl", "linux/386")
+			targetBinary := "linux/386"
```

Comment:
we should really just be using the arch of the cluster.

I thought there was some work to do this already with images, though I can't remember where it is (or if it's been merged yet, even). @luxas do you recall?